packages/fluent-ui/src/WrapIfAdditionalTemplate/WrapIfAdditionalTemplate.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
```

Comment:
For consistency with the other themes, did you want to move this into the `FieldTemplate` directory?

Reply:
Also, this is the bug that needs to be fixed for this https://github.com/rjsf-team/react-jsonschema-form/issues/2777

Reply:
Will move `WrapIfAdditionalTemplate` into its own directory for each theme and update changelog to reference #2777